src/server/config/schema.js

```diff
@@ -146,7 +145,9 @@ module.exports = () => Joi.object({
       bounds: Joi.array().items(Joi.array().items(Joi.number()).min(2).required()).min(2)
     }).default()
   }).default(),
-
+  maps: Joi.object({
```

Comment:
> that service array could contain other map types down the road as well.

True, but let's not get ahead of ourselves. `maps` is pretty generic, and I don't think we want to actually deprecate the `tilemap` config. It's a totally valid alternative to using our tileservice.